README.md

```diff
@@ -1 +1,42 @@
-# OOP-school-library
\ No newline at end of file
+# OOP-school-library
+
+A Ruby library to manage a school library using latest technologies.
+
+## Built With
+
+- Major languages: Ruby
+- Technologies used: Git, Github and Gitflow,
+
+## Getting Started
+
+### Setup
+- git clone https://github.com/juanmanuelbyc/OOP-school-library.git
+
+## Authors
+
+👤 Juan Diaz (Juan Diaz)
+
+- GitHub: [@juanmanuelbyc](https://github.com/juanmanuelbyc)
+- Twitter: [@twitter](https://twitter.com/juanmanueldiar)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-díaz-5281b3111/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/juanmanuelbyc/OOP-school-library/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] OPTIONAL
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.